internal/vsp/feepayment.go

```diff
@@ -731,7 +736,29 @@ func (fp *feePayment) reconcilePayment() error {
 	if feeTx == nil || len(feeTx.TxOut) == 0 {
 		err := fp.makeFeeTx(nil)
 		if err != nil {
-			fp.schedule("reconcile payment", fp.reconcilePayment)
+			var apiErr *BadRequestError
+			if errors.As(err, &apiErr) {
+				if apiErr.Code == codeTicketCannotVote {
+					fp.remove("ticket cannot vote")
+					// Attempt to Revoke Tickets, we're not returning any errors here
+					// and just logging.
+					n, err := w.NetworkBackend()
+					if err != nil {
+						log.Errorf("unable to get network backend for revoking tickets %v", err)
+					} else {
+						if rpc, ok := n.(*dcrd.RPC); ok {
+							err := w.RevokeTickets(ctx, rpc)
+							if err != nil {
+								log.Errorf("cannot revoke vsp tickets %v", err)
+							}
+						} else {
+							log.Errorf("cannot revoke ticket since SPV mode")
```

Comment:
we can remove this entirely or perhaps make it a debug log; this isn't an error.